T145/react-challenge-project-jan-2022
# Challenge #5

## Changes
1. Fixed the "Logout" button in the navbar.

## Purpose
The "Logout" button would only redirect to the login page and not logout a user. https://github.com/Shift3/react-challenge-project-jan-2022/issues/5

## Approach
_How does this change address the problem?_

## Pre-Testing TODOs
N/A

## Testing Steps
Log in as a valid user, then log out. Submit orders as the user and not as the user to verify that it works.

## Learning
*Improvement:*
- https://react-redux.js.org/using-react-redux/connect-mapdispatch

Closes https://github.com/Shift3/react-challenge-project-jan-2022/issues/5
